in_toto/common_args.py

```diff
@@ -195,6 +195,13 @@ def parse_password_and_prompt_args(args):
     "help": ("generate metadata using dsse (experimental)."),
 }
 
+RUN_TIMEOUT_ARGS = ["--run-timeout"]
+RUN_TIMEOUT_KWARGS = {
+    "dest": "run_timeout",
+    "default": 10,
+    "help": ("float that represents the max timeout for the in-toto-run command"),
```

Comment:
This should specify it's in seconds.

Reply:
And tbh, should we even support sub-second values? I don't know that we have use cases for this value that require such accuracy.

Reply:
+1 on full seconds